### DISCUSSION TITLE
Client Install Script to apply to all users in system on Ubuntu & Windows

### DISCUSSION THREAD

**Discussion by mdtabrezpasha** [rustdesk/rustdesk#7231] [Q&A]
Hello everyone,
                         we have a network with more than 800 systems and each system has two accounts, one is admin account and another one is a FreeIPA account,  settings get applied to the admin account but not to the freeipa account.  Has someone deployed the client  on a system with mutiple accounts?

**Comment by rustdesk** [comment]
This https://rustdesk.com/docs/en/self-host/client-deployment/ is system wide, not user wide.
@dinger1986 could you confirm?

**Reply by dinger1986** [reply]
yes system wide

**Comment by mdtabrezpasha** [comment]
i checked it again, it's applying to local users only, not FreeIPA users.

**Reply by dinger1986** [reply]
possibly something to do where /home is for the FreeIPA users, not something I have used

**Comment by mdtabrezpasha** [comment]
I am trying another method to see if that works, that is to fork rustdesk
repo and create custom installers.. followed all the instructions, but
getting the installer with default settings.

**Comment by mdtabrezpasha** [comment]
the install log showing this.. it installs the program but doesn't generate the configuraiton files.

tee: /home//.config/rustdesk/RustDesk2.toml: No such file or directory
chown: cannot access '/home//.config/rustdesk/RustDesk2.toml': No such file or directory
grep: /home//.config/rustdesk/RustDesk.toml: No such file or directory
grep: /home//.config/rustdesk/RustDesk.toml: No such file or directory",

**Reply by dinger1986** [reply]
Well there you go, you can see it it's not finding the username so the folder doesn't exist, this is more hope FreeIPA creates users I think

**Comment by mdtabrezpasha** [comment]
The same issue is occurring for local users also.

**Reply by dinger1986** [reply]
Ok but it's not finding the home directory

**Comment by mdtabrezpasha** [comment]
It's all there in the file system.. I can do regular file operations, just
this script doesn't work. It creates the rustdesk folder only under root
user's config folder
